recipes/creme/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "creme" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/creme-ml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "109e308070e023825efe0b3eb6f94898e59a24ce52da259828b2eb473dfed4c6"
+
+build:
+  skip: True  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.16.4
+    - pip
+    - python
+    - scikit-learn >=0.21.2
+    - scipy >=1.3.0
+  run:
+    - numpy >=1.16.4
+    - python
+    - scikit-learn >=0.21.2
+    - scipy >=1.3.0
+
+test:
+  imports:
+    - creme
+    - creme.cluster
+    - creme.compat
+    - creme.compose
+    - creme.decomposition
+    - creme.ensemble
+    - creme.feature_extraction
+    - creme.feature_selection
+    - creme.impute
+    - creme.linear_model
+    - creme.metrics
+    - creme.multiclass
+    - creme.multioutput
+    - creme.naive_bayes
+    - creme.optim
+    - creme.optim.losses
+    - creme.plot
+    - creme.preprocessing
+    - creme.proba
+    - creme.reco
+    - creme.stats
+    - creme.tree
+    - creme.utils
+
+about:
+  home: "https://github.com/creme-ml/creme"
+  license: "BSD"
```

Which BSD licence is it? At at glance it looks like this but please check
```suggestion
  license: BSD-3-Clause
```

---

it's bsd-3, will change